source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -41,6 +41,55 @@ _For more on linting (including standard settings and config) see the [linting][
 * Lint checks using [PyFlakes][]
 
 
+### Dependencies
+
+It is important to have a good strategy for specifying your project's Python
+libraries. There are two desirable characteristics:
+
+1. Reproducibility
+
+    Pin your full dependencies or you'll get unpredictability between your dev
+    environment and other environments. You want a new starter to avoid small
+    hard-to-spot problems. And you want parity between what you test locally,
+    what is tested by CI and what you deploy, or you risk new issues appearing
```

Comment:
Going to have to nitpick for an Oxford comma here please: 'what is tested by CI, and what you deploy' 😛

Reply:
👍